package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "app",
+  "private": true,
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
```

Comment:
- [ ] At the moment, the app is not starting because webpack is not part of the dependencies. Please kindly ensure that all dependencies are available and that you can run it successfully locally